tests/test_utils.py

```diff
@@ -892,3 +892,83 @@ def test_bucket_is_valid_ok():
     valid, message = utils.bucket_is_valid(bucket_name="something-.")
     assert valid
     assert message == ""
+
```

Comment:
We should add a test to check that the float is ok for a huge file size and a huge number of seconds (> 3 months)

Reply:
Just tested with:
size_stored: 1000000000000000
hours_stored: 2928.58 (~4 months)

and it seems fine:

```
dds_backend      | [2022-08-30 09:06:11,453] utils [DEBUG] 2928.581792627222
dds_backend      | [2022-08-30 09:06:11,454] scheduled_tasks [INFO] Version: <File Version 1>	Usage: 2.928581792627222e+18
dds_backend      | [2022-08-30 09:06:11,455] scheduled_tasks [INFO] Project someunit00001 byte hours: 2.928581792627222e+18
```
```
┏━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━┓
┃ Project ID    ┃ Title          ┃ PI                  ┃ Status      ┃ Last updated                  ┃    Size ┃   Usage ┃      Cost ┃ Access ┃
┡━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━┩
│ someunit00001 │ test           │ valyo@me.com        │ In Progress │ Tue, 30 Aug 2022 08:31:17 UTC │ 445.0 B │ 2.9 EBH │ 366.1 Kkr │ ✅     │
│ project_1     │ First Project  │ support@example.com │ In Progress │ Tue, 30 Aug 2022 08:24:05 UTC │   0.0 B │  0.0 BH │    0.0 kr │ ✅     │
│ project_2     │ Second Project │ support@example.com │ In Progress │ Tue, 30 Aug 2022 08:24:05 UTC │   0.0 B │  0.0 BH │    0.0 kr │ ✅     │
```

Reply:
I also changed test_calculate_bytehours_ok accordingly